### ISSUE TITLE
Allow providing command line options to GitHub action

### ISSUE THREAD

**Issue by tburrows13** [psf/black#1751]
Currently the GitHub action (https://github.com/psf/black/blob/master/Dockerfile) is hardcoded to use the arguments `--check --diff`. It would be useful to be able to provide custom options, in particular the target Python version.

I used to use https://github.com/lgeiger/black-action, which allowed any command line arguments to be passed through - unfortunately it stopped working this week, so I found this official action instead.

**Comment by tburrows13** [comment]
Sorry, I only just noticed #1755, which has now been merged - it looks great, but I had a couple more points:

1. That PR updates `docs/github_actions.md` to mention the new functionality, but the same information is repeated in the readme: https://github.com/psf/black/blob/master/README.md#github-actions, which has not been updated.

2. Would it be possible to make this repo appear in the GitHub marketplace, so that when you search https://github.com/marketplace?query=black it shows up. I (and presumably many others) only found out that there was an 'official' black action from a random comment somewhere on GitHub.
